hazelcast/src/test/java/com/hazelcast/config/XMLConfigBuilderTest.java

```diff
@@ -407,6 +407,8 @@ public void readEurekaConfig() {
                 + "            <eureka enabled=\"true\">\n"
                 + "                <use-public-ip>true</use-public-ip>\n"
                 + "                <namespace>hazelcast</namespace>\n"
+                + "                <shouldUseDns>false</shouldUseDns>\n"
+                + "				   <serviceUrl.default>http://localhost:8082/eureka</serviceUrl.default>\n"
```

Comment:
Could you please add an assertion for the two added properties and do the same changes in the `YamlConfigBuilderTest` too?

Reply:
Done